code/headers/uarm_swift_pro.hpp

```diff
@@ -16,19 +16,20 @@ namespace r2d2::robot_arm {
         const uint16_t default_speed = 1000;
 
     private:
+        using usart_c = r2d2::usart::usart_connection_c;
         uarm_gcode_generator_c<50> gcode_generator;
-        hardware_usart_c usart_bus;
+        usart_c &usart_bus;
 
     public:
         /**
          * Robot arm constructor, needs an usart bus for communication
          */
-        uarm_swift_pro_c(const r2d2::hardware_usart_c &usart_bus);
+        uarm_swift_pro_c(usart_c &usart_bus);
         /**
          * robot arm constructor, creates a usart bus from the bautrate and port
          **/
-        uarm_swift_pro_c(unsigned int &bautrate,
-                         r2d2::uart_ports_c &usart_port);
+        //  uarm_swift_pro_c(unsigned int &bautrate,
+        //                 r2d2::usart::uart_ports_c &usart_port);
```

Comment:
commented uit code will not be approved when merging with the master.

Reply:
fixed in https://github.com/R2D2-2019/robot_arm/commit/654c3de55e13a57ff32a750c14c4098c7a6bed7b